packages/nextjs/app/admin/_components/SubmissionTabs.tsx

```diff
@@ -28,6 +28,9 @@ export const SubmissionTabs = ({ submissions }: { submissions: Submission[] }) =
     },
   );
 
+  const votedLabel = connectedAddress ? `Voted (${voted.length})` : "Voted (0)";
+  const notVotedLabel = connectedAddress ? `Not Voted (${notVoted.length})` : "Not Voted (0)";
```

Comment:
Since the "else" are loading states, what about using `(-)` instead of `(0)`

Reply:
Since it was the only thing I went ahead and did it https://github.com/BuidlGuidl/extensions-hackathon/pull/68/commits/3ba22e666add9e2c698fec84f96e00b17af80216